src/pages/Possibility/KivanSlider.vue

```diff
@@ -0,0 +1,371 @@
+			<kv-icon
+				name="small-chevron"
+				class="left-arrow"
+			/>
+		</button>
+		<kv-carousel-slide
+			v-for="(slide , index) in slidesData"
+			:key="`image-${index}`"
+			:class="`slide-${index}`"
+			:transition-name="transitionName"
+		>
+			<div class="row">
+				<div class="small-12 large-6">
+					<kv-responsive-image :images="slide.left" />
+				</div>
+				<div class="small-12 large-6 show-for-large">
+					<kv-responsive-image :images="slide.right" />
+				</div>
+			</div>
+		</kv-carousel-slide>
+		<button @click="nextSlide">
+			<kv-icon
+				name="small-chevron"
+				class="right-arrow"
+			/>
+		</button>
+	</kv-carousel>
+</template>
+
+<script>
+import KvResponsiveImage from '@/components/Kv/KvResponsiveImage';
+import KvCarousel from '@/components/Kv/KvCarousel';
+import KvCarouselSlide from '@/components/Kv/KvCarouselSlide';
+import KvIcon from '@/components/Kv/KvIcon';
+
+
+const slidesImageRequire = require.context('@/assets/images/possibilities-banners/kivan-slider', true);
+
+export default {
+	components: {
+		KvResponsiveImage,
+		KvCarousel,
+		KvCarouselSlide,
+		KvIcon,
+	},
+	data() {
+		return {
+			slidesData:
+			[
+				{
+					left: [
+						['small', slidesImageRequire('./kivan-1a-std.jpg')],
+						['small retina', slidesImageRequire('./kivan-1a-retina.jpg')]
+					],
+					right: [
+						['small', slidesImageRequire('./kivan-1b-std.jpg')],
+						['small retina', slidesImageRequire('./kivan-1b-retina.jpg')],
+					]
+				},
+				{
+					left: [
+						['small', slidesImageRequire('./kivan-2a-std.jpg')],
+						['small retina', slidesImageRequire('./kivan-2a-retina.jpg')],
+					],
+					right: [
+						['small', slidesImageRequire('./kivan-2b-std.jpg')],
+						['small retina', slidesImageRequire('./kivan-2b-retina.jpg')],
+					]
+				},
+				{
+					left: [
+						['small', slidesImageRequire('./kivan-3a-std.jpg')],
+						['small retina', slidesImageRequire('./kivan-3a-retina.jpg')],
+					],
+					right: [
+						['small', slidesImageRequire('./kivan-3b-std.jpg')],
+						['small retina', slidesImageRequire('./kivan-3b-retina.jpg')]
+					]
+				}
+			],
+			currentIndex: 0,
+			transitionName: 'kv-slide-left'
+		};
+	},
+	methods: {
+		previousSlide() {
+			this.transitionName = 'kv-slide-right';
+			this.$nextTick(() => {
+				this.$refs.KvCarousel.previous();
+			});
+		},
+		nextSlide() {
+			this.transitionName = 'kv-slide-left';
+			this.$nextTick(() => {
+				this.$refs.KvCarousel.advance();
+			});
+		},
+		slideTransition(payload) {
+			this.currentIndex = payload;
+		},
+		setSlide(index) {
+			this.$refs.KvCarousel.goToSlide(index);
+		}
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import "settings";
+
+$cta-color: #611b1f;
+$cta-color2: #00244e;
+$cta-color3: #02582e;
+
+.kv-carousel {
+	margin-bottom: rem-calc(140);
+
+	.left-arrow,
+	.right-arrow {
+		z-index: 10000;
+		stroke: $white;
+		height: rem-calc(30);
+		width: rem-calc(30);
+		position: absolute;
+		top: 29%;
```

Comment:
We should be able to handle top without multiple media queries by using some calc() magic since we know the height of our arrows. Something along the lines of top: calc(50% - #{rem-calc(15)}).